.github/workflows/image.yml

```diff
@@ -31,14 +32,18 @@ jobs:
           platforms: linux/${{ matrix.platform }}
           dockerfile_path: "./Dockerfile"
           build_args: SERVICE_NAME=${{ matrix.service }}
+          build_target: ${{ matrix.target }}
           ghcr: true
-          tag_suffix: -${{ matrix.platform }}
+          tag_suffix: -${{ matrix.platform }}${{ matrix.target == 'prod' && '' || '-dev' }}
```

Comment:
We can add `publish_on_pr: true` to test to see if this works before merging

Reply:
Also, should we update devservices to use the dev version?